plugins/modules/zos_tso_command.py

```diff
@@ -111,6 +111,10 @@
            - LISTDSD DATASET('HLQ.DATA.SET') ALL GENERIC
       max_rc: 4
 
+- name: Execute TSO command to run explicitly a REXX script from a data set.
```

Comment:
Are all the words necessary? It's not wrong or extraneous per se, 
Could simplify to:
> Execute TSO command to run a REXX script from a data set.

Or, to slightly reword what you've got
>name: Execute TSO command to explicitly run a REXX script from a data set.

Swap the order of `explicitly` and `run`

Reply:
There is a distinction between running a REXX script explicitly and implicitly: https://www.ibm.com/docs/en/zos/3.1.0?topic=exec-running
And the example uses a fully qualified data set outside of a system file, so it is running explicitly.